src/components/PostList/PostList.jsx

```diff
@@ -1,3 +1,10 @@
-export const PostList = () => (
-  <>Put the list here</>
+import './PostList.scss';
+import { PostInfo } from '../PostInfo';
+
+export const PostList = ({ posts }) => (
+  <div className="PostList">
+    {posts.map(post => (
+      <PostInfo post={post} key={post.id} />
```

Comment:
```suggestion
      <PostInfo post={post}  />
```
or here. ( best way remove inside component*)

Reply:
Misha said that the first tag after .map should contain key.
```suggestion
      <PostInfo post={post} key={post.id} />
```
It's the first tag after .map
So I remove it inside component and left it here